### DISCUSSION TITLE
wintenApps compatibility win10 21H2 / NVDA 2023.1

### DISCUSSION THREAD

**Discussion by TLG95** [josephsl/wintenApps#89] [Q&A] [answered]
Hi,

Is there a win10 21H2 compatible version for NVDA 2023.1 please ?

**Comment by josephsl** [comment]
Here it is: https://github.com/josephsl/wintenApps/releases/download/23.08/wintenApps-23.05.7.nvda-addon

**Reply by TLG95** [reply]
Thank you for your reply.
I have already tried to install this version of the extension but I always encounter this message: You are using Windows 10 21H2 (19044), a version of Windows not supported by this extension

**Comment by josephsl** [comment]
Hi, I see. Try this one instead: https://github.com/josephsl/wintenApps/releases/download/23.05/wintenApps-23.05.nvda-addon

**Reply by TLG95** [reply]
Hi, same result

**Comment by josephsl** [comment, marked as answer]
Hi, I see. I usually do not test Windows 10 feature updates once support for Home and Pro versions are no more (sorry), I assume this is a computer running Windows 10 21H2 Enterprise or Education, and if this is Windows 10 Enterprise, would it be possible to ask your organizational IT folks to upgrade to 22H2? The story changes if you are using Windows 10 Enterprise LTSC 2021, a version not supported by the add-on. Thanks.

**Reply by TLG95** [reply]
It is indeed a Windows 21H2 Enterprise version. The update to 22H2 will be done at the end of the year.
Thank you for your answers
